### PR TITLE
CRM-21055 Revert change of "cancel" to "close" on backend contribution form

### PR DESCRIPTION
**Pull request by agh1**
This reverts commit 35c930c1921ad45ae3da24b2e0bdad7470ed7b49.

The change was done in isolation, leading to inconsistency with other forms
that have the Save / Save and New / Cancel pattern.

@totten @colemanw please take a look before shipping 4.7.25--I just noticed #10845 as I was writing up the release notes and I'm very concerned about it.  See [the explanation on JIRA](https://issues.civicrm.org/jira/browse/CRM-21055?focusedCommentId=108600&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-108600)

Basically, either all instances of this pattern should say "Close" or all should say "Cancel".  In the meantime, we shouldn't be shipping it with the one exception, especially since it changes longstanding behavior.

@eileenmcnaughton @pradpnayak @monishdeb @lcdservices

### CODE REVIEW

**Comment by JoeMurray** [comment]
I don't think this should be hastily reverted. See issue for discussion.

**Comment by eileenmcnaughton** [comment]
Per comments on Chat I AM going for a hasty revert. I don't think this was done hastily as it has been in the rc for months, or without discussion. However, I think it only provides a trivial benefit and trivially affects lots of people so caution is the better part of valour here

**Comment by eileenmcnaughton** [comment]
hmm looks like we missed the release!

**Comment by totten** [comment]
There seems to be general support for reverting the label change. Joe's point is good that it wouldn't justify reissuing a release on its own -- but we're doing a reissue anyway due to the release-notes.

**Comment by eileenmcnaughton** [comment]
Note that this might yet get re-introduced - but I think we need to discuss further first since Andrew had some good points (to be reconciled with good points made by others in favour of this change)
